### ISSUE TITLE
Error loading with VS2017 v15.8

### ISSUE THREAD

**Issue by rkhoury** [MBulli/SmartCommandlineArgs#64]
I get this error in the Commandline Arguments window:

> An exception was encountered while constructing the content of this frame.  This information is also logged in "C:\Users\richardk\AppData\Roaming\Microsoft\VisualStudio\15.0_2c79d5f3\ActivityLog.xml".
> 
> Exception details:
> System.Exception: Exception of type 'System.Exception' was thrown.
>    at Microsoft.VisualStudio.Shell.Interop.IVsShell5.LoadPackageWithContext(Guid& packageGuid, Int32 reason, Guid& context)
>    at Microsoft.VisualStudio.Platform.WindowManagement.WindowFrame.GetPackage()
>    at Microsoft.VisualStudio.Platform.WindowManagement.WindowFrame.ConstructContent()

Is that anything related to this plugin needing updating with newer VS verisons, or something the issue with my local environment?  (also, how do I go about debugging these kinds of issues myself with a local checkout of this extension?)

Many thanks,
Richard.

**Comment by MBulli** [comment]
We‘re currently on a trip and will check that issue as soon as we‘re back. 

If I have to guess we maybe need to implement AsyncPackage or something like that. I already done that on a local branch but hadn‘t the time to extensively test it and push it. 

Debugging the extension should be straightforward. Just clone this repo and hit run (ignore all unit tests). Make sure to run the experimental instance of vs and to build wirh Debug. Maybe vs ask you to install the vs sdk.

**Comment by rkhoury** [comment]
Thanks for the super-quick response!  I've not done any VS extension programming (and certainly debugging!) so I'll see how I go testing this out :)

**Comment by rkhoury** [comment]
Another observation of this error: this doesn't appear to happen in solutions I have with VC toolset being used, it only appears in a solution for another platform.
Anyway, I'll hopefully get a chance to debug something later...

**Comment by MBulli** [comment]
Are you using clang? We never tried any platform besides MSVC maybe we need to make some changes for other platforms.

**Comment by rkhoury** [comment]
I'm not sure what went wrong, but only after loading my other-platform solution through a debugging session of this extension did everything correct itself.
